### PR TITLE
fix: issue #414

### LINKED ISSUE #414: Can't call setEmergencyExit after calling revokeStrategy
Use case:

I call `revokeStrategy()` so strategy returns funds and after not getting everything I call `setEmergencyExit()` to get everything out.

### PR DESCRIPTION
**Pull request by poolpitako**
Fixes #414

### CODE REVIEW

**Comment by fubuloubu** [comment]
It looks like you did some swaps of tabs and spaces

**Comment by poolpitako** [comment]
> It looks like you did some swaps of tabs and spaces

you can add `w=1` to the diff and it will remove whitespace diff.

**Comment by steffenix** [comment]
I think we shouldn't commit space changes.

**Review by steffenix — APPROVED** [review]

File: contracts/Vault.vy [hunk 1]
```diff
@@ -1421,7 +1421,8 @@ def revokeStrategy(strategy: address = msg.sender):
     @param strategy The Strategy to revoke.
     """
     assert msg.sender in [strategy, self.governance, self.guardian]
-    assert self.strategies[strategy].debtRatio != 0 # dev: already zero
+    if self.strategies[strategy].debtRatio == 0:
+        return # no-op
```
Comment (fubuloubu) [inline contracts/Vault.vy:1425]:
```suggestion
    # NOTE: This function may be called via `BaseStrategy.setEmergencyExit` while the
    #       strategy might have already been revoked or had the debt limit set to zero
    if self.strategies[strategy].debtRatio == 0:
        return # already set to zero, nothing to do
```

**Review by fubuloubu — APPROVED** [review]
All the whitespace changes made this a little harder to review. Next time please put formatting updates in another PR

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**